content/04.network_applications.md

```diff
@@ -102,23 +102,23 @@ Knowledge graphs have been applied to many biomedical challenges ranging from id
 In this section we review how knowledge graphs are applied in biomedical settings and put particular emphasis on an emerging set of techniques that represent knowledge graphs in a low dimensional space.
 
 ![
-Overview of biomedical applications that make use of knowledge graphs.
+Overview of various biomedical applications that make use of knowledge graphs.
 Categories consist of: (a) Multi-Omic applications, (b) Pharmaceutical Applications and (c) Clinical Applications.
 ](images/figures/unifying_applications_overview.png){#fig:unifying_applications}
 
 #### Multi-Omic Applications
 
 Multi-omic applications employ knowledge graphs to study the genome, how genes are expressed in the transcriptome, and how the products of those transcripts interact in the proteome.
 These graphs are used to establish connections between -omic entities as well as diseases.
-Tasks in this context include gene-symptom prioritization [@doi:10.1093/jamia/ocy117], protein-protein interaction prediction [@doi:10.1089/cmb.2012.0273; @doi:10.1186/1752-0509-9-S1-S9], and detecting miRNA-disease associations [@doi:10.1155/2017/2498957].
+Tasks in this context include gene-symptom prioritization [@doi:10.1093/jamia/ocy117], protein-protein interaction prediction [@doi:10.1089/cmb.2012.0273; @doi:10.1186/1752-0509-9-S1-S9] and detecting miRNA-disease associations [@doi:10.1155/2017/2498957].
 We focus specifically on multi-omic applications that represent knowledge graphs in a low dimensional space to make connections.
 
 Recommendation systems make use of knowledge graphs to establish links between RNA with disease and proteins with other proteins.
 Shen et al. used an algorithm called collaborative filtering to establish an association between miRNA and diseases [@doi:10.1155/2017/2498957].
-The authors constructed an miRNA-Disease network using the Human MicroRNA Disease database (HMDD) [@doi:10.1093/nar/gky1010] and generated an adjacency matrix with the rows representing miRNA and the columns representing diseases.
+The authors constructed a miRNA-Disease network using the Human MicroRNA Disease database (HMDD) [@doi:10.1093/nar/gky1010] and generated an adjacency matrix with the rows representing miRNA and the columns representing diseases.
```

Comment:
I think this might actually be `an mRNA...` because the soft `m` is pronounced. Do you know the rule here? I'm not sure.

Reply:
This was a hard one. It can go either way; however, using the [google scholar search method](https://www.aje.com/arc/editing-tip-indefinite-article-use-abbreviations/) the most common form uses "a" instead of "an".  Will default to "a".

Statistics provided if curious:
"a miRNA" has 69,200 hits
"an miRNA" has 12,500 hits

Reply:
Wow! Today I learned! 🤔  Thanks :)